Subject: [PATCH] Fix net/core/sock.o build failure
Date:	Thu, 30 Dec 2004 23:21:16 +0000
X-BK-ChangeSetKey: acme@conectiva.com.br[torvalds]|ChangeSet|20041230232116|56152
Patch-mainline: yes
From: olh@suse.de


ChangeSet 1.2082, 2004/12/30 15:21:16-08:00, acme@conectiva.com.br

	[PATCH] Fix net/core/sock.o build failure
	
	This fixes a build failure that happens when you don't select IPV6.
	
	Signed-off-by: Linus Torvalds <torvalds@osdl.org>



 ipv6.h |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)


diff -Nru a/include/linux/ipv6.h b/include/linux/ipv6.h
--- a/include/linux/ipv6.h	2004-12-30 19:45:18 -08:00
+++ b/include/linux/ipv6.h	2004-12-30 19:45:18 -08:00
@@ -273,6 +273,7 @@
 	struct ipv6_pinfo inet6;
 };
 
+#if defined(CONFIG_IPV6) || defined(CONFIG_IPV6_MODULE)
 static inline struct ipv6_pinfo * inet6_sk(const struct sock *__sk)
 {
 	return inet_sk(__sk)->pinet6;
@@ -283,7 +284,6 @@
 	return &((struct raw6_sock *)__sk)->raw6;
 }
 
-#if defined(CONFIG_IPV6) || defined(CONFIG_IPV6_MODULE)
 #define __ipv6_only_sock(sk)	(inet6_sk(sk)->ipv6only)
 #define ipv6_only_sock(sk)	((sk)->sk_family == PF_INET6 && __ipv6_only_sock(sk))
 #else
-
To unsubscribe from this list: send the line "unsubscribe bk-commits-head" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

